m2c/asm_file.py

```diff
@@ -548,6 +554,12 @@ def process_label(label: str, *, kind: LabelKind) -> None:
                         data = parse_incbin(args, options, warnings)
                         if data is not None:
                             asm_file.new_data_bytes(data)
+                    elif directive == ".obj":  # decomp-toolkit label format
```

Comment:
btw, maybe move this up next to the handling for .rel which is also related to decomp-toolkit?